CHANGELOG.md

```diff
@@ -12,14 +16,32 @@
 - Added support for custom B2C domains
 - Improved MSAL error handling
 
+## [0.3.4] - 2019-03-07
+### Fixed
+- Improve logging for token removal scenarios
+- Use ASCII for PKCE code challenge
+- Don't return Access token if ID token/Account are missing
+
+## [0.3.3] - 2019-05-29
+### Updated
+- Ignore cached fields in JSON if they contains "null"
+
+## [0.3.2] - 2019-05-24
+### Updated
+- Updated to newer v2 broker protocol version
+
 ## [0.3.1] - 2019-05-06
 ### Fixed
 - Better error handling in CBA cancellation flows
 - Don't read corrupted refresh tokens from cache
 
 ## [0.3.0] - 2019-04-22
 ### Added
+<<<<<<< HEAD
```

Comment:
this conflict hasn't been addressed yet :P

Reply:
yes, fixed :)